### PR TITLE
Run optimized p0 tests

### PR DESCRIPTION
**Pull request by eapolinario**
# TL;DR
Run optimized P0 test suite and also non-P0 tests

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by eapolinario** [comment]
Why aren't tests running?

**Comment by eapolinario** [comment]
will incorporate those changes to boilerplate.
